pom.xml

```diff
@@ -12,16 +12,15 @@
 	</parent>
 	<properties>
 		<java.version>1.7</java.version>
-		<kafka.version>0.10.1.1</kafka.version>
-		<spring-kafka.version>1.1.2.RELEASE</spring-kafka.version>
-		<spring-integration-kafka.version>2.1.0.RELEASE</spring-integration-kafka.version>
+		<kafka.version>0.10.2.1</kafka.version>
+		<spring-kafka.version>1.2.0.RELEASE</spring-kafka.version>
```

Comment:
A cursory test shows that this works with the previous versions (and we do test with Kafka 0.10.2 separately). So we may just want to put them back - not the SI-Kafka version though.